utils/recursiveDependencyFetch.js

```diff
@@ -34,9 +35,8 @@ const extractDependencies = (input, pkg) => {
 };
 
 const packageJsonUrl = path => {
-  const [_full, name, version] = path.match(
-    /https:\/\/unpkg.com\/(@?[^@\n]*)@?(\d+\.\d+\.\d+)?/
-  );
+  const { name, version } = parseUrl(path);
```

Comment:
Good callout for the nullable comment in the description. Right now we return `undefined` implicitly if we don't match something in `parseUrl` and that will break right here.

If we _do_ decide to return undefined we should do something like:

```js
const { name, version } = parseUrl(path) || {};
if (!name) {
  console.error(`Unable to parse path: ${path}`);
  return;
}